### PR TITLE
add segment bool to send events to second segment acct

### PR DESCRIPTION
**Pull request by melvinsoft**
Configuration Pull Request
---
This PR is a follow up of: https://github.com/appsembler/edx-platform/pull/284/

It just adds a new var to the auth json, to allow us enable sending events to a second segment account.

Make sure that the following steps are done before merging

  - [ ] @devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?  
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Review by OmarIthawi — APPROVED** [review]
LGTM. Thanks @melvinsoft!

**Comment by OmarIthawi** [comment]
@melvinsoft the tests are failing. Which is weird, since it succeeds on: https://github.com/appsembler/configuration/pull/180

Any idea why?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -447,6 +447,7 @@ EDXAPP_OPTIMIZELY_PROJECT_ID: "None"
 EDXAPP_TRACKING_SEGMENTIO_WEBHOOK_SECRET: ""
 EDXAPP_CMS_SEGMENT_KEY: "None"
 EDXAPP_LMS_SEGMENT_KEY: "None"
+EDXAPP_SEGMENT_SITE: "None"
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/defaults/main.yml]:
```suggestion
EDXAPP_ENABLE_SEGMENT_SITE: False
```

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -1055,6 +1056,7 @@ generic_env_config:  &edxapp_generic_env
 lms_auth_config:
   <<: *edxapp_generic_auth
   SEGMENT_KEY: "{{ EDXAPP_LMS_SEGMENT_KEY }}"
+  SEGMENT_SITE: "{{ EDXAPP_SEGMENT_SITE }}"
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/defaults/main.yml:1059]:
```suggestion
  SEGMENT_SITE: "{{ EDXAPP_ENABLE_SEGMENT_SITE }}"
```

File: playbooks/roles/edxapp/defaults/main.yml [hunk 3]
```diff
@@ -1128,6 +1130,7 @@ lms_env_config:
 cms_auth_config:
   <<: *edxapp_generic_auth
   SEGMENT_KEY: "{{ EDXAPP_CMS_SEGMENT_KEY }}"
+  SEGMENT_SITE: "{{ EDXAPP_SEGMENT_SITE }}"
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/defaults/main.yml]:
```suggestion
  SEGMENT_SITE: "{{ EDXAPP_ENABLE_SEGMENT_SITE }}"
```

**Review by OmarIthawi — APPROVED** [review]
Thanks @melvinsoft!

Would be less confusing with the changes below.

The tests are failing. Which is weird, since it succeeds on: #180

Any idea why?

**Review by thraxil — APPROVED** [review]
Approve with @OmarIthawi's suggested changes.

**Pull request merged**